routes/signout.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const router = express.Router();
+
+router.post("/", (req, res) => {
+  res.clearCookie("rds-session");
```

Comment:
The cookie name should be extracted in a constant (if not done already) and that constant should be used here

Reply:
Ok on it. Will make changes

Reply:
const cookie = req.cookies;
res.clearCookie(cookie); 

how does this look?

Reply:
As per the latest commit I resolved this issue please check it and let me know if it can be resolved